wdtk-datamodel/src/main/java/org/wikidata/wdtk/datamodel/interfaces/DataObjectFactory.java

```diff
@@ -254,9 +268,27 @@ QuantityValue getQuantityValue(BigDecimal numericValue,
 	 *            the unit of this quantity, or the empty string if there is no
 	 *            unit
 	 * @return a {@link QuantityValue} corresponding to the input
+	 * @deprecated use {@link #getQuantityValue(BigDecimal, BigDecimal, BigDecimal, ItemIdValue)}
```

Comment:
@wetneb I've got some warnings after merging this. Could you please look at it? Firstly, `@deprecated` in javadoc and `@Deprecated` annotation should be used together. Secondly, existing calls (e.g. the one in `DatamodelConverter` must use the new API as otherwise they will trigger warnings.

Reply:
Thanks for noticing, see #594